frontend/src/metabase/visualizations/visualizations/Table.jsx

```diff
@@ -130,10 +130,14 @@ export default class Table extends Component {
             data: { cols, rows },
           },
         ],
-        settings,
+        { "table.pivot": pivotCol },
```

Comment:
`table.pivot` is a boolean, I think you want `table.pivot_column`.

We should probably have some tests of the settings themselves. There are a couple in `visualization.unit.spec.js`